application/views/admin/sessions.php

```diff
@@ -372,9 +375,28 @@
             alertify.<?= $t ?>("<?= $m ?>");
 <?php endif; ?>
 
-    $('#sessions_table').on('click', '.reload-attendee', function () {
+    $('.reload-attendee').on('click', '.reload-attendee', function () {
```

Comment:
@slive3718  Why did you change `#sessions_table` to `.reload-attendee` ?   This was added to hook the `onclick` event work even after pagination in `DataTable`